manifests/agent/config.pp

```diff
@@ -2,7 +2,9 @@
 
   Ini_setting {
     path    => $puppet::conf,
+    section => 'agent',
     ensure  => 'present',
+    notify  => Service['puppet_agent'],
```

Comment:
Not following sorry. If we are using cron or none, this notify does nothing (schedules a refresh for the service, but as long as the service is down then nothing will ever happen)